public/app.js

```diff
@@ -8,15 +8,15 @@ var app = app || {};
     Object.keys(projectDataObject).forEach(key => this[key] = projectDataObject[key]);
   }
 
-  Project.prototype.toHtml = () => {
+  Project.prototype.toHtml = function() {
     var theTemplateScript = $('#article-template').html();
     var theTemplate = Handlebars.compile(theTemplateScript);
     var theCompiledHtml = theTemplate(this);
     $('#portfolio').append(theCompiledHtml);
   }
 
   Project.buildProjects = (data) => {
-    data.forEach(function(item){
+    data.map((item) => {
```

Comment:
Better to use `forEach` if you don't need to use the return value.

You're already satisfying the `map` requirement [here](https://github.com/bbeeksma/Portfolio/blob/deb1421ace4ab9195e631fe5ceaa2293109f1ab5/public/scripts/models/project.js#L20).